pkg/scheduler/factory_test.go

```diff
@@ -205,6 +207,7 @@ func TestCreateFromConfig(t *testing.T) {
 				PreBind: schedulerapi.PluginSet{Enabled: []schedulerapi.Plugin{{Name: "VolumeBinding"}}},
 				Bind:    schedulerapi.PluginSet{Enabled: []schedulerapi.Plugin{{Name: "DefaultBinder"}}},
 			},
+			wantErr: "",
```

Comment:
no need to explicitly set this, the default is empty string. ditto in the other places below.